service/router.go

```diff
@@ -56,6 +57,12 @@ func SigningRouter() *mux.Router {
 	router.Handle("/v1/pivotmodel", Middleware(ErrorHandler(pivot.ModelAssertion))).Methods("POST")
 	router.Handle("/v1/pivotserial", Middleware(ErrorHandler(pivot.SerialAssertion))).Methods("POST")
 
+	// Test log upload routes (only in the factory)
+	if datastore.Environ.Config.Driver == "sqlite3" {
```

Comment:
Does it mean that the mode cloud vs factory is recognized based on the database driver being used?

Reply:
Yes, not ideal, but the only way that I can see to identify if we're in the factory. The form doesn't have any CSRF protection, to make it simpler to use from the device test tools, so it's not something that we should deploy in the cloud.